python/paddle/fluid/layers/detection.py

```diff
@@ -1638,67 +1638,102 @@ def prior_box(input,
               name=None,
               min_max_aspect_ratios_order=False):
     """
-    **Prior Box Operator**
-
-    Generate prior boxes for SSD(Single Shot MultiBox Detector) algorithm.
+    This op generates prior boxes for SSD(Single Shot MultiBox Detector) algorithm.
     Each position of the input produce N prior boxes, N is determined by
     the count of min_sizes, max_sizes and aspect_ratios, The size of the
     box is in range(min_size, max_size) interval, which is generated in
     sequence according to the aspect_ratios.
 
-    Args:
-       input(Variable): The Input Variables, the format is NCHW.
-       image(Variable): The input image data of PriorBoxOp,
-            the layout is NCHW.
-       min_sizes(list|tuple|float value): min sizes of generated prior boxes.
-       max_sizes(list|tuple|None): max sizes of generated prior boxes.
+    Parameters:
+       input(Variable): 4-D tenosr(NCHW), the data type should be float32 or float64.
```

Comment:
NCHW  -> in shape [N, C, H, W], N is the batch size ........

Reply:
hmm...I think it's too redundancy, so I remove the explanation